compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -49,13 +47,10 @@ import org.w3c.dom.events.CompositionEventInit
 import org.w3c.dom.events.Event
 
 private class InputInteractor(
```

Comment:
I'd suggest folding this into a `TextInputTestScope` or something and providing a `runTextInputTest`. Like I did in my POC branch.

Reply:
I got rid of InputInteractor completely